src/test/java/com/buyanywhere/productcatalog/dto/CategoryDtoTest.java

```diff
@@ -0,0 +1,100 @@
+package com.buyanywhere.productcatalog.dto;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+public class CategoryDtoTest {
+
+    private CategoryDto categoryDto;
+
+    @Before
+    public void setup(){
+        categoryDto = new CategoryDto();
+        categoryDto.setId((long) 1);
+        categoryDto.setName("Home");
+        categoryDto.setDisplayOrder(1);
+    }
+
+    @Test
+    public void getId_shouldReturnId(){
```

Comment:
If you perform a test on getters and setters without any logic in it, then you are testing that the language is doing what it was programmed to do, hence you are not testing your code but the complier's and interpreter's codes wich is not the point of TDD